src/extension.ts

```diff
@@ -94,10 +112,27 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	return Promise.resolve(context);
 }
 
+export async function _stopExistingDfaRun(context: vscode.ExtensionContext, outputChannel: vscode.LogOutputChannel, verbose: boolean) {
```

Comment:
Boolean arguments are typical code smells. 

I see that there is only one place where you are passing in false... in that case - I wonder if you should really be calling this method or if there should be a separate helper method for that case. Either in the other comment, or right here... can you explain why you are calling the stop existing dfa run method during activation? Is it simply to call through to 
context.globalState.update(Constants.GLOBAL_DFA_PROCESS, undefined);

If so, can we either just put that in the activate step... or put what you need for that in a separate method and try to keep each method having a single responsibility?